example/src/index.ts

```diff
@@ -223,6 +229,14 @@ function main(): void {
   },
   {
     selector: '.jp-Cell.jp-mod-commandMode',
+    sequence: ['C'],
+    handler: () => {
+      nbWidget.copy();
+      return true;
```

Comment:
I think `return true;` might be superfluous:

https://github.com/phosphorjs/phosphide/blob/master/src/shortcutmanager/plugin.ts#L176